jujugui/static/gui/src/app/assets/js/dev.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+window.juju_config = require('./config');
+window.YUI = require('../../../../build/app/assets/javascripts/yui/yui/yui').YUI;
```

Comment:
Parcel wraps everything so it has to be imported rather than using a `<script>` tag. Same for `juju_config` above.